qiskit_machine_learning/kernels/statevector_kernel.py

```diff
@@ -0,0 +1,190 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""Statevector Quantum Kernel"""
+
+from __future__ import annotations
+
+import numpy as np
+from qiskit import QuantumCircuit
+from qiskit.quantum_info import Statevector
+
+from .base_kernel import BaseKernel
+
+
+class StatevectorKernel(BaseKernel):
```

Comment:
Naming question, should we go for `StatevectorQuantumKernel` for consistency with `FidelityQuantumKernel`?? I normally prefer shorter names, but you know, the usual discussion.

Reply:
Yeah, I think that's more consistent as you say, and am happy to change it to that.

Reply:
As discussed elsewhere, I've now renamed the class `FidelityStatevectorKernel`.

Reply:
> As discussed elsewhere, I've now renamed the class `FidelityStatevectorKernel`.

Thanks!